h/session.py

```diff
@@ -89,6 +89,10 @@ def _group_model(route_url, group):
         model['url'] = route_url('group_read',
                                  pubid=group.pubid,
                                  slug=group.slug)
+
+    if user is not None and user.id == group.creator_id:
```

Comment:
So this does two things that I think we'd like to avoid: it links the concept of a group creator with the concept of a group moderator outside the group model, and it makes explicit reference to primary keys present on the model.

I think it might be equally convenient and more future-friendly to have the Group model expose a `moderators` property which returns a list of users. For now, the implementation can be as simple as:

```@property
def moderators(self):
    return [self.creator]
```

And then the check here would be:

```if user is not None and user in group.moderators:```

Which would a) make the intent much clearer, and b) extend straightforwardly to groups having 0 or multiple moderators.